lib/logstash/inputs/stomp.rb

```diff
@@ -59,6 +75,10 @@ def register
       connect
       subscription_handler # is required for re-subscribing to the destination
     }
+
+    # Display configuration information
+    @logger.info("Reconnect is disabled, plugin will not reconnect on connection error") if !@reconnect
+    @logger.info("Debug is disabled, turn on debug for full exceptions") if !@debug
```

Comment:
You can use `--verbose` or `--debug` flags when launching logstash. These flags are mentioned in https://www.elastic.co/guide/en/logstash/current/command-line-flags.html